src/declarations.d.tsx

```diff
@@ -0,0 +1,5 @@
+// This entire file will be deleted once we have typescript definitions
```

Comment:
The typescript definitions for esm-api are published to npm now -- we can delete this comment.